multiqc/modules/bcftools/stats.py

```diff
@@ -112,45 +128,56 @@ def parse_bcftools_stats(self):
                     'anchor': 'bcftools-stats',
                     'content': linegraph.plot(self.bcftools_stats_indels, pconfig)
                 })
+            # Make line graph of variants per depth
+            if len(depth_data) > 0:
+                import pprint
+                pprint.pprint(depth_data)
+                pconfig = {'id': 'bcftools_stats_depth', 'title': 'Variant depths',
+                           'ylab': 'Fraction of sites (%)', 'xlab': 'Variant depth',
+                           'ymin': 0, 'ymax': 100, 'categories': True}
+                self.sections.append({'name': 'Variant depths',
+                                      'anchor': 'bcftools-stats',
+                                      'content': linegraph.plot(depth_data, pconfig)})
 
         # Return the number of logs that were found
         return len(self.bcftools_stats)
 
     def bcftools_stats_genstats_table(self):
         """ Add key statistics to the General Stats table """
         stats_headers = OrderedDict()
-        stats_headers['number_of_SNPs'] = {
-            'title': 'M SNPs',
-            'description': 'Number of SNPs (millions)',
-            'min': 0,
-            'format': '{:.2f}',
-            'modify': lambda x: x / 1000000
+        stats_headers['number_of_records'] = {
```

Comment:
Ah ok, this is a good point. Fine to leave it then I guess - could check the max value for all samples, but I guess it would be pretty confusing if it varied between reports.

I try to avoid commas where possible because of the German / Swedish / some-other-European-countries habit of using commas to denote decimal places (an even more annoying problem when parsing data). But I could make it use the same formatting as HighCharts with spaces (`5 522 770`). Already have the back end to customise this too: http://multiqc.info/docs/#number-formatting